modules/aws/elbv2.go

```diff
@@ -0,0 +1,127 @@
+package aws
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/elbv2"
+	"github.com/stretchr/testify/require"
+)
+
+// LoadBalancer is an Amazon load balancer.
+type LoadBalancer struct {
+	Name                  string // The name of the load balancer.
+	ARN                   string // The Amazon Resource Name (ARN) of the load balancer.
+	CanonicalHostedZoneID string // The ID of the Amazon Route 53 hosted zone associated with the load balancer.
+	DNSName               string // The public DNS name of the load balancer.
+}
+
+// CreateLoadBalancerFromElbV2 creates a LoadBalancer struct from provided elbv2.LoadBalancer
+func CreateLoadBalancerFromElbV2(elbv2 *elbv2.LoadBalancer) (lb *LoadBalancer) {
```

Comment:
I am unsure. It looks like to be a helper that translate one format into another. If you really, want to go this path, you should make it private and don't expose as an API function.

Reply:
In a perfect world, your test case should not interact with already created resources. It's better to create a resource you need for the test. You will make your test more independent. 

I don't think, you will need to have `Describe*` functions here. I would vote for Create* and Delete*. Once that done, you won't need `CreateLoadBalancerFromElbV2`.

Reply:
Finally, start with small things, and once you expose your PR, ask maintainers if they're interested in additional functionality.